File: examples/mvp-java/src/main/java/co/windly/limbosample/presentation/PresentationModule.java
package co.windly.limbosample.presentation;

import co.windly.limbosample.presentation.main.MainModule;
import co.windly.limbosample.presentation.splash.SplashModule;
import dagger.Module;
import dagger.android.support.AndroidSupportInjectionModule;

@Module(includes = {
  AndroidSupportInjectionModule.class,
  SplashModule.class,
  MainModule.class
})
public class PresentationModule {
  // No-op.
}

File: samples/kotlin/src/main/java/co/windly/limbosample/presentation/PresentationModule.kt
package co.windly.limbosample.presentation

import co.windly.limbosample.presentation.main.MainModule
import co.windly.limbosample.presentation.splash.SplashModule
import dagger.Module
import dagger.android.AndroidInjectionModule

@Module(includes = [
  AndroidInjectionModule::class,
  MainModule::class,
  SplashModule::class
])
class PresentationModule
